storage/rethinkdb/bootstrap.go

```diff
@@ -0,0 +1,137 @@
+package rethinkdb
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/dancannon/gorethink"
+)
+
+func makeDB(session *gorethink.Session, name string) error {
+	_, err := gorethink.DBCreate(name).RunWrite(session)
+	if err != nil {
+		if strings.Contains(err.Error(), "already exists") {
+			return nil
+		}
+
+		return err
+	}
+
+	resp, err := gorethink.DB(name).Wait().Run(session)
```

Comment:
Comments is good for code review but not a good way to remind people to update this function when upgrade the rethink to a higher version.

But it won't be a big problem either, since the build/test will fail if it has not been updated when using the higher version of rethink.